cmd/boulder-ca/main.go

```diff
@@ -176,7 +176,7 @@ func main() {
 		conn, err := bgrpc.ClientSetup(c.CA.PublisherService, scope)
 		cmd.FailOnError(err, "Failed to load credentials and create connection to service")
 		cai.Publisher = bgrpc.NewPublisherClientWrapper(pubPB.NewPublisherClient(conn), c.CA.PublisherService.Timeout.Duration)
-	} else {
+	} else if amqpConf.Publisher != nil {
```

Comment:
No, because the CA doesn't currently use gRPC to talk to the Publisher, right? It uses AMQP.